.release/ci.hcl

```diff
@@ -0,0 +1,198 @@
+schema = "1"
+
+project "nomad" {
+  team = "nomad"
+  slack {
+    // #proj-nomad-releases
+    notification_channel = "CUYKT2A73"
+  }
+  github {
+    organization     = "hashicorp"
+    repository       = "nomad"
+    release_branches = ["main"]
+  }
+}
+
+event "merge" {
+  // "entrypoint" to use if build is not run automatically
+  // i.e. send "merge" complete signal to orchestrator to trigger build
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository   = "nomad"
+    workflow     = "build"
+  }
+}
+
+event "upload-dev" {
+  depends = ["build"]
+  action "upload-dev" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "upload-dev"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "security-scan-binaries" {
+  depends = ["upload-dev"]
+  action "security-scan-binaries" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "security-scan-binaries"
+    config       = "security-scan.hcl"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "notarize-darwin-amd64" {
+  depends = ["security-scan-binaries"]
+  action "notarize-darwin-amd64" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "notarize-darwin-amd64"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "notarize-darwin-arm64" {
+  depends = ["notarize-darwin-amd64"]
+  action "notarize-darwin-arm64" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "notarize-darwin-arm64"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "notarize-windows-386" {
+  depends = ["notarize-darwin-arm64"]
+  action "notarize-windows-386" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "notarize-windows-386"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "notarize-windows-amd64" {
+  depends = ["notarize-windows-386"]
+  action "notarize-windows-amd64" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "notarize-windows-amd64"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
```

Comment:
Can these notarization steps run in parallel? Right now they all depend on each other, which makes it a serial process. I remember Apple notarization being particularly slow sometimes 😅

Reply:
@lgfa29 they currently have to be run sequentially - but we do have plans to support parallelization in the future

Reply:
Thanks! I think this would speed-up the process quite a bit 🙂